webui/react/src/stores/users.tsx

```diff
@@ -52,6 +56,13 @@ class UserStore extends PollingStore {
     });
   }
 
+  protected getLoadablePaginationByParams(params: GetUsersParams = {}) {
```

Comment:
question: do we need `getUsersPagination`. and `getLoadablePaginationByParams`? both look the same func. and if we keep both `getLoadablePaginationByParams` could be private func

Reply:
yeah that's true, probably could just keep the one `getUsersPagination`